docs/concepts/pdp/configuration.mdx

```diff
@@ -0,0 +1,346 @@
+
+Default: `1`
+
+The timeout for querying OPA for an allow decision, in seconds. `0` means no timeout.
+
+#### PDP_OPA_CLIENT_FAILURE_THRESHOLD
+
+Default: `0.1`
+
+The percentage of failed requests to OPA that will trigger a failure, causing the healthcheck to fail.
+
+#### PDP_OPA_CLIENT_FAILURE_THRESHOLD_INTERVAL
+
+Default: `60`
+
+The interval to calculate the failure threshold, in seconds.
+
+#### PDP_OPA_CONFIG_FILE_PATH
+
+Default: `~/opa/config.yaml`
+
+The path on the container for OPA config file.
+
+#### PDP_OPA_AUTH_POLICY_FILE_PATH
+
+Default: `~/opa/basic-authz.rego`
+
+The path on the container for OPA authorization policy (rego file).
+
+#### PDP_OPA_BEARER_TOKEN_REQUIRED
+
+Default: `True`
+
+If true, all API calls to OPA must provide a bearer token (the value of CLIENT_TOKEN).
+
+#### PDP_OPA_DECISION_LOG_ENABLED
+
+Default: `True`
+
+If true, OPA decision logs will be uploaded to the Permit.io cloud console.
+
+#### PDP_OPA_DECISION_LOG_CONSOLE
+
+Default: `False`
+
+If true, OPA decision logs will also be printed to console (only relevant if `OPA_DECISION_LOG_ENABLED` is true).
+
+#### PDP_OPA_DECISION_LOG_INGRESS_ROUTE
+
+Default: `/v2/decision-logs/ingress`
+
+The route on Permit.io the decision logs will be uploaded to.
+
+#### PDP_OPA_DECISION_LOG_INGRESS_BACKEND_TIER_URL
+
+Default: `https://decision-log-ingress.api.permit.io`
+
+The URL that decision logs will be uploaded to.
+Automatically configured by the control plane.
+
+#### PDP_OPA_DECISION_LOG_MIN_DELAY
+
+Default: `1`
+
+The minimum amount of time (in seconds) to wait between decision log uploads.
+
+#### PDP_OPA_DECISION_LOG_MAX_DELAY
+
+Default: `10`
+
+The maximum amount of time (in seconds) to wait between decision log uploads.
+
+#### PDP_OPA_DECISION_LOG_UPLOAD_SIZE_LIMIT
+
+Default: `65536`
+
+Log upload size limit in bytes. OPA will chunk uploads to cap message body to this limit.
+
+#### PDP_ALLOW_METRICS_UNAUTHENTICATED
+
+Default: `False`
+
+If true, the `/metrics` endpoint will be accessible without authentication.
+
+#### PDP_LOCAL_FACTS_WAIT_TIMEOUT
+
+Default: `10`
+
+When using SDK proxy facts via PDP, the default time to wait for the facts to be available, in seconds.
+
+
+## OPAL Configurations
+
+:::note
+The list below includes only the configurations that are most commonly used.
+Additional configurations are available in the [OPAL documentation](https://docs.opal.ac/getting-started/configuration).
+:::
+
+#### OPAL_SERVER_URL
+
+Default: `https://opal-v2.permit.io`
+
+The URL of the OPAL server.
+Automatically configured by the control plane.
+
+
+#### OPAL_SERVER_WS_URL
+
+Default: `wss://opal-v2.permit.io`
+
+The WebSocket URL of the OPAL server.
+Automatically configured by the control plane.
+
+#### OPAL_CLIENT_TOKEN
+
+Default: automatically configured
+
+The client token used to authenticate the PDP against the OPAL server.
+Automatically configured by the control plane.
+
+#### OPAL_INLINE_OPA_ENABLED
+
+Default: `True`
```

Comment:
I'm not sure about the purpose of this documentations as you can't really configure them ( and even if you can you shouldn't because it will break things )
@obsd @orweis WDYT ?